website/project/model.py

```diff
@@ -860,6 +860,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     primary_institution = fields.ForeignField('institution')
 
     affiliated_institutions = fields.ForeignField('institution', list=True)
+    visit = fields.IntegerField(default=0)
```

Comment:
That seems like a fragile connection, but I guess it makes withdrawing something from a conference easy.

Rather than a script to update this every day, it might be better to add a `StringField` indicating whether the node is connected to a conference (`'<conf_name>'` if so, `None` or `''` if not), and then increment `.visit` in the file view function when the submitted materials are viewed iff `node.conference_submission`.